library/modules/Persistence.cpp

```
@@ -291,6 +374,54 @@ static bool load_file(const std::filesystem::path & path, int entity_id) {
     return true;
 }
 
+static bool load_persist_files(const std::filesystem::path& path)
+{
+    Json::Value json;
+    try {
+        std::ifstream file(path);
+        file >> json;
+    }
+    catch (std::exception&) {
+        // empty file?
+        return false;
+    }
```

At least log the exception reason as a diagnostic; failing silently is unkind